source/__tests__/integration/profile-create.test.js

```diff
@@ -0,0 +1,38 @@
+const path = require('path')
+
+describe('Sinopia Profile Editor imports a Profile', () => {
+
+  beforeAll(async () => {
+    await page.goto('http://127.0.0.1:8000/#/profile/create')
+  })
+
+  describe('Import Data Modal Upload Dialog', () => {
+
+    beforeAll(async () => {
+      await expect(page).toClick('a', { text: 'Import'})
+    })
+
+    it('click the import button and display dialog', async () => {
```

Comment:
Is this better worded as "displays Import Data dialog" as the clicking is happening in beforeAll?